pkg/lib/registry/registry.go

```diff
@@ -128,23 +128,40 @@ func unpackImage(ctx context.Context, reg image.Registry, ref image.Reference) (
 
 func populate(ctx context.Context, loader registry.Load, graphLoader registry.GraphLoader, querier registry.Query, reg image.Registry, refs []image.Reference, mode registry.Mode, overwrite bool) error {
 	unpackedImageMap := make(map[image.Reference]string, 0)
+	expectedBundles := make(map[string]registry.Package)
+	overwrittenBundles := make(map[string][]string)
```

Comment:
In this case `make` is semantically equivalent to `map[string]registry.Package{}`